platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/tipset/TipsetEventCreatorImpl.java

```diff
@@ -207,15 +207,15 @@ public GossipEvent maybeCreateEvent() {
             return createEventForSizeOneNetwork();
         }
 
-        final long bullyScore = tipsetWeightCalculator.getMaxBullyScore();
-        tipsetMetrics.getBullyScoreMetric().update(bullyScore);
+        final long selfishness = tipsetWeightCalculator.getMaxSelfishnessScore();
+        tipsetMetrics.getSelfishnessMetric().update(selfishness);
 
-        // Never bother with anti-bullying techniques if we have a bully score of 1. We are pretty much guaranteed
-        // to bully ~1/3 of other nodes by a score of 1.
-        final double beNiceToNerdChance = (bullyScore - 1) / antiBullyingFactor;
+        // Never bother with anti-selfishness techniques if we have a selfishness score of 1.
+        // We are pretty much guaranteed to selfishness ~1/3 of other nodes by a score of 1.
```

Comment:
You need a form of `selfishness` that functions as a verb. `ignore`?

Reply:
```
        // Never bother with anti-selfishness techniques if we have a selfishness score of 1.
        // We are pretty much guaranteed to selfish to ~1/3 of other nodes by a score of 1.
        final double beNiceChance = (selfishness - 1) / antiSelfishnessFactor;
```